### ISSUE TITLE
loss is 0 and grad_norm is nan

### ISSUE THREAD

**Issue by zzksdu** [Ola-Omni/Ola#12]
i get a issue like [loss is 0 and grad_norm is nan] when i try to reproduce your pretrain result in stage1. my setting is as follows:

**EXP_NAME="ola_7b_stage1_1_llava_oneVision_webui_0227"
DATA="xxx/LLaVA_OneVision_Data_laion.json" 

CHECKPOINT='xxx/common_models/Qwen2___5-7B-Instruct' 
VISION_TOWER="xxx/common_models/google__siglip-so400m-patch14-384"**

torchrun \
    --nnodes=1 \
    --node_rank=0 \
    --master_addr=127.0.0.2 \
    --nproc_per_node=${GPUS} \
    --master_port=${MASTER_PORT} \
    ola/train/train.py \
    --deepspeed ./scripts/zero2.json \
    --run_name $EXP_NAME \
    --model_name_or_path $CHECKPOINT \
    --vision_tower $VISION_TOWER \
    --mm_projector_type ola_mlp \
    --mm_vision_select_layer -1 \
    --mm_use_im_patch_token False \
    --tune_speech_adapter False \
    --version qwen_1_5 \
    --data_path $DATA \
    --bf16 True \
    --output_dir xxxx/$EXP_NAME \
    --sample_independently True \
    --fix_speech_encoder True \
    --freeze_mm_vision_tower True \
    --speech_encoder "xxx/Ola_speech_encoders/large-v3.pt" \
    --music_encoder "xxx/Ola_speech_encoders/BEATs_iter3_plus_AS2M_finetuned_on_AS2M_cpt2.pt" \
    --speech_encoder_type "dual" \
    --speech_encoder_hidden_size 2048 \
    --speech_encoder_ds_rate 10 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 1000 \
    --save_total_limit 1 \
    --learning_rate 1e-3 \
    --weight_decay 0.0 \
    --warmup_ratio 0.05 \
    --min_lr_ratio 0.01 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 16384 \
    --gradient_checkpointing True \
    --dataloader_num_workers 8 \
    --frames_upbound 48 \
    --lazy_preprocess True \
    --report_to none
the log is as follows:

![Image](https://github.com/user-attachments/assets/e517e11f-bbc3-4ddb-ba74-ee070e532672)
do you have any suggestion to this issue? thanks @liuzuyan @dongyh20

**Comment by ztw1123** [comment]
你好，我也遇到了这个问题，想问一下您是怎么解决的呢？
